tests/foreman/ui/test_lifecycleenvironment.py

```diff
@@ -152,7 +148,7 @@ def test_positive_search_lce_content_view_packages_by_full_name(session, module_
 
 @pytest.mark.tier3
 @pytest.mark.skipif((not settings.robottelo.REPOS_HOSTING_URL), reason='Missing repos_hosting_url')
-def test_positive_search_lce_content_view_packages_by_name(session, module_org):
+def test_positive_search_lce_content_view_packages_by_name(session, module_org, target_sat):
```

Comment:
Optional: Since target_sat is already included here, a `session` here can also be replaced by `target_sat.ui_session()`

Reply:
Good suggestion but I think thats out of the scope of this PR !